ui/public/locales/en.json

```diff
@@ -3184,6 +3188,7 @@
 "message.success.disable.saml.auth": "Successfully disabled SAML authorization",
 "message.success.disable.vpn": "Successfully disabled VPN",
 "message.success.edit.acl": "Successfully edited ACL rule",
+"message.success.edit.nfsopts": "Successfully updated NFS options",
```

Comment:
May be here ! mentioning about the effect of changes after cancelling the maintenance mode ?

Reply:
Have moved Edit NFS mount options to Edit Primary Storage. So this message was removed.